packages/patternfly-4/src/content/get-started/gettingstarted-design.md

```diff
@@ -2,7 +2,7 @@
 path: "/get-started/designers"
 ---
 # Getting started for designers
```

Comment:
I think this is OK for now, but could be made much more robust. Maybe some of the materials being developed now for training can be dropped in here. Also, there is no mention of the Sketch library.

Reply:
Agree, although I dont think we can mention the sketch library yet bc it's not ready to be made available - that's the feedback I got from Gina and Kyle.